packages/hardhat/contracts/FundingVaultFactory.sol

```diff
@@ -37,14 +39,24 @@ contract FundingVaultFactory{
     error FundingVaultFactory__deadlineCannotBeInThePast();
```

Comment:
Remove suffixes.

Reply:
Fixed: FundingVaultFactory__ is removed in latest commit